### PR TITLE
Pam config fixes

### PR DESCRIPTION
**Pull request by fsateler**
Add pam_unix account module to systemd-user pam snippet and enable skipping install of the same snippet.

### CODE REVIEW

File: DISTRO_PORTING [hunk 1]
```diff
@@ -48,6 +48,9 @@ PAM:
         /etc/security/limits.conf will not be read unless you load pam_limits.
         Make sure you add modules your distro expects from user services.
 
+	Pass --with-pamconfdir=no to ./configure installing this file and
```
Comment (martinpitt) [inline DISTRO_PORTING]: s/installing/to not install/ ?

File: src/login/systemd-user.m4 [hunk 2]
```diff
@@ -2,6 +2,8 @@
 #
 # Used by systemd --user instances.
 
+account required pam_unix.so
```
Comment (poettering) [inline src/login/systemd-user.m4:5]: If ew add pam_unix in for "account", shouldn't we also add it in for the others? 

(i don't know want to suggest we should, I am just wondering this myself)

File: configure.ac [hunk 3]
```diff
@@ -1510,6 +1510,7 @@ AC_ARG_WITH([pamconfdir],
         AS_HELP_STRING([--with-pamconfdir=DIR], [Directory for PAM configuration]),
```
Comment (poettering) [inline configure.ac]: help text should be updated, i figure

File: src/login/systemd-user.m4 (hunk 2 above)
Comment (fsateler) [reply src/login/systemd-user.m4:5]: `pam_unix(8)` says:

> The session component of this module logs when a user logins or leave the system.

Not very relevant for our purposes.

Also, I don't see why the password and auth modules are relevant. `systemd --user` has no way to ask the user for feedback (eg, change or input the password).

**Comment by fsateler** [comment]
Force pushed an updated version, except for adding the other module types for pam_unix

File: src/login/systemd-user.m4 (hunk 2 above)
Comment (poettering) [reply src/login/systemd-user.m4:5]: ok

**Comment by poettering** [comment]
lgtm

**Pull request merged**